website/client/components/userMenu/inbox.vue

```diff
@@ -209,6 +209,16 @@
     color: $gray-200;
     margin-bottom: 0.5rem;
   }
+
+  .messagePreview {
+    display: block;
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
```

Comment:
can you remove the -webkit prefixes from rule names? it'll be added automatically at compile time

Also it doesn't look like firefox and IE11 support it... is it really needed? @Tressley 

I'm wondering if we could get the same result by using `overflow: hidden;` and setting the height of the box given that we know the height of each line

Reply:
I've read that `-webkit` is not supported by some browsers, but I found a similar solution and thought it's normal.

https://github.com/HabitRPG/habitica/blob/4289becccc8663985736094e9fa2360d44733f0c/website/client/components/challenges/challengeItem.vue#L45-L60

But if you say this isn't a good solution, I'll fix up it.
(But I still don't know how to implement an ellipsis using `overflow: hidden;` :( If it(ellipsis)'s definitely necessary, I'll try again)

Reply:
Ah, i didn't noticed it was used elsewhere , in that case it's fine as long as we make sure the layout isn't too broken on Firefox for example

Reply:
Sorry, just getting a chance to answer a few pings; I do think it's necessary for this to work in Firefox, but not IE.